### ISSUE TITLE
Baozi Manhua - Inverted comics list. 

### ISSUE THREAD

**Issue by derbysig** [tachiyomiorg/tachiyomi-extensions#14894]
### Source information and language

Baozi Manhua 1.3.17

### Steps to reproduce

Steps to reproduce
Select a comic (e.g. The Daily Life of the Immortal King)
Select a chapter that is not titled Chapter 1
Scroll through the chapter into the following chapter

### Expected behavior

Expectation 1: After Step 1, the chairs should load in reverse chronological order with the newest chapter at the top and the oldest chapter at the bottom

### Actual behavior

Actual 1: The chapters load in reverse, even though the date information is correct.

Actual 2: After completing a chapter, it goes to the previous chapter numerically. Finishing chapter 2 takes your to chapter 1 instead of 3

### Tachiyomi version

0.14.3

### Android version

Android 13

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.14.3](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by stevenyomi** [comment]
I think it's just because you set the chapter sorting to ascending within the app. Check if your chapter sorting is **"↓ By source"**.

**Comment by derbysig** [comment]
I tried multiple settings but it's still inverted. 
All 3 showed same behaviour.
Currently found that it affects these comics:
- Immortal King's Daily Life
- Tales of Demon and Gods
![Screenshot_20230112-160114](https://user-images.githubusercontent.com/122437585/212011470-e65aa9ff-60ce-4787-9cb1-3e316926fa0e.png)
![Screenshot_20230112-160106](https://user-images.githubusercontent.com/122437585/212011478-84cd4250-1d49-467c-8f9a-74bb5e556a80.png)

**Comment by derbysig** [comment]
I tried multiple settings but it's still inverted. 
All 3 showed same behaviour.
Currently found that it affects these comics:
- Immortal King's Daily Life
- Tales of Demon and Gods
![Screenshot_20230112-160114](https://user-images.githubusercontent.com/122437585/212011470-e65aa9ff-60ce-4787-9cb1-3e316926fa0e.png)
![Screenshot_20230112-160106](https://user-images.githubusercontent.com/122437585/212011478-84cd4250-1d49-467c-8f9a-74bb5e556a80.png)

**Comment by stevenyomi** [comment]
It's an issue related to Traditional Chinese site, similar to  #14313. I'll fix it now.
